ci/pkl-pipelines/common/shared_resources.pkl

```diff
@@ -168,4 +168,39 @@ class AssumeRunCodeBuildRoleStep extends Pipeline.TaskStep {
   }
 }
 
-LoadRunCodeBuildAssumedRoleVar: Pipeline.LoadVarStep = loadVarJson("role", "run-codebuild-assume-role/assume-role.json")
\ No newline at end of file
+LoadRunCodeBuildAssumedRoleVar: Pipeline.LoadVarStep = loadVarJson("role", "run-codebuild-assume-role/assume-role.json")
+
+class ParseGithubAlphaReleaseTagTask extends Pipeline.TaskStep {
+  hidden gitRelease: String
+
+  task = "parse-release-tag"
+  file = "pay-ci/ci/tasks/parse-release-tag.yml"
+  input_mapping {
+    ["git-release"] = gitRelease
+  }
+}
+
+class ParseECRCandidateTagTask extends Pipeline.TaskStep {
+  hidden ecr_repo: String
+
+  task = "parse-candidate-tag"
+  file = "pay-ci/ci/tasks/parse-candidate-tag.yml"
+  input_mapping {
+    ["ecr-repo"] = ecr_repo
+  }
+}
+
+class AssumeConcourseRoleTask extends Pipeline.TaskStep {
+  hidden aws_account_name: AWSAccountName = "test"
+  hidden output_name: Pipeline.Identifier = "assume-retag-role"
+
+  task = "assume-retag-role"
+  file = "pay-ci/ci/tasks/assume-role.yml"
+  output_mapping = new {
+    ["assume-role"] = output_name
+  }
+  params = new {
+    ["AWS_ROLE_ARN"] = "arn:aws:iam::((pay_aws_\(aws_account_name)_account_id)):role/concourse"
+    ["AWS_ROLE_SESSION_NAME"] = "retag-ecr-image"
```

Comment:
Do we rely on this session name value anywhere? (Prior to the refactor it was `retag-ecr-image-as-release`)

Reply:
No, they are purely informational and I don't actually think anyone has ever read it